deploy.md

```diff
@@ -0,0 +1,68 @@
+## Deployment Notes ##
+
+### Login ###
+To login to the NIA account as ad administrator from CLI:
+
+```
+aws sts assume-role --role-arn arn:aws:iam::{deployment account}:role/{deployment role} --role-session-name  admin@nia --serial arn:aws:iam::{identity account}:mfa/{username} --token-code {mfa code}
+export AWS_ACCESS_KEY_ID=RoleAccessKeyID
+export AWS_SECRET_ACCESS_KEY=RoleSecretKey
+export AWS_SESSION_TOKEN=RoleSessionToken
+```
+
+This is implemented in [get_aws_key.sh]. As this script sets environment variables you'll need to `source get_aws_key.sh` to persist these in your session. 
+
+Duration is set to the default for this role. If this times out you will need to unset the environment variables to request a new session:
+
+```
+unset AWS_ACCESS_KEY_ID AWS_SECRET_ACCESS_KEY AWS_SESSION_TOKEN
+```
+
+This is implemented in [unset_aws_key.sh].
+
+### Adapting Docker Compose into ECS tasks ###
```

Comment:
I haven't given these instructions a try, but I'm guessing they're useful?!  They feel less related to deployment, so I'm not sure `deploy.md` is the right place.